index.html

```diff
@@ -58,6 +58,50 @@
 					</aside>
 				</section>
 
+				<!-- Pettry dish -->
+				<section data-background="http://www.bionetsa.com/uploads/3/6/2/3/3623525/7098366_orig.jpg">
+
+					<aside class="notes">
+						זה משחק של מספרים.
+					</aside>
+				</section>
+
+				<!-- Mass customization -->
+				<section data-background="http://www.bionetsa.com/uploads/3/6/2/3/3623525/7098366_orig.jpg">
+					<span style="background-color: black; opacity: 0.85; padding: 20px 40px;">
+						Mass customization
+					</span>
+				</section>
+
+
+				<!-- Investor -->
+				<section data-background="https://cloud.githubusercontent.com/assets/125707/12099080/dd46f2e8-b32e-11e5-8190-26457bbf96b8.jpg">
+					<div style="color: white;font-size: 80%; background-color: black; opacity: 0.85; padding: 20px 40px;">
+						<div>1,000,000$</div>
+						<div>---------------</div>
+						<div>==> 13 projects * ~70,000$</div>
```

Comment:
@bricel @niryariv fyi, I've updated the slide with approximate numbers we've recalculated